sdk/communication/communication-administration/test/utils/recordedClient.ts

```diff
@@ -71,6 +77,45 @@ export function createRecordedCommunicationIdentityClient(
   };
 }
 
+export function createRecordedCommunicationIdentityClientWithToken(
+  context: Context
+): RecordedClient<CommunicationIdentityClient> {
+  const recorder = record(context, environmentSetup);
+
+  if (isBrowser()) {
```

Comment:
I'm not following why we have to use connection string in a browser environment instead of managed identities

Reply:
We now skip the test in the case the token is undefined